tests/keras/layers/test_advanced_activations.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+from numpy.testing import assert_allclose
+import numpy as np
+from keras import backend as K
+
+np.random.seed(1337)
```

Comment:
one more thing. Putting seed into the file like that is not going to give you deterministic results. Test frameworks (pytest or unittest) collect files first and then run the tests. During file collection this code will be executed but since file collection in general not a deterministic process (e.g. in multiprocessing setup), you'll end up with some random seed before actual test run. I understand that its verbose, but unfortunately you should better put seed into each test with rand.